docs/data-collection/preliminary.md

```diff
@@ -1,16 +1,128 @@
+
+The overall experimental setting can be sumarized as follows:
+
+``` mermaid
+flowchart TB
+
+    subgraph "Scanning Room"
+        direction TB
+        sr1[Scanner]
+        sr2[Cannula]
+        sr3[RB]
+        sr4[ECG]
+        sr5[ET]
+    end
+
+    subgraph "BIOPAC"
+        direction TB
+        biopac1[DA100C]
+        biopac2[ECG100C MRI]
+        biopac3[AMI100D]
+        biopac4[STP100D]
+    end
+
+    sr1 --->|Trigger| sb[Syncbox]
+    sr2 ---> ga[Gas Analyzer]
+    sr3 --->|"Negative (-)"| biopac1[DA100C]
+    sr4 ---> biopac2[ECG100C MRI]
+    sr5 ---> et[Eye Tracker PC]
+
+    ga --->|"Channel 3 (CO<sub>2</sub>)"| biopac3
+    ga --->|"Channel 4 (O<sub>2</sub>)"| biopac3
+
+    sb --->|USB| pc1["Stimuli presentation Laptop ({{ secrets.hosts.psychopy | default("███") }})"]
+    pc1 <--->|Ethernet| et
+    pc1 --->|USB| modem[MMBT-S Interface]
+    pc1 --->|HDMI| display[Proyector]
+    modem --->|25-pin parallel| biopac4
+
+    BIOPAC --->|Ethernet| pc2["Physio-recording Laptop ({{ secrets.hosts.acqknowledge | default("███") }})"]
+```
+
+The above graph can be broken down as follows:
+
+1. **Signals generating from the Scanning Room**.
+    In addition to the MR imagery produced by the scanner, the participant will be wearing the RB, the ECG leads, a nasal cannula to retrieve the expired gases, and finally their right eye will be recorded with the infrared camera of the ET.
+    All those probes and devices carry signals outside the Scanning Room either through cables or tubes, with the access panel as the interface.
+    The access panel also has a connector carrying the **trigger** signals generated by MR schemes, which indicate important events in the MRI acquisition (typically, one trigger pulse is generated for each new *repetition time* —TR—, e.g., with every fMRI volume).
+1. **Syncbox**.
+    A *NordicLabs Syncbox* receives TTL (transistor-transistor logic) triggers from the scanner.
+    This box can just forward the triggers converted into other formats and/or manipulate them (e.g., filter, generate, etc.).
+1. **Gas analyzer (GA)**.
+    The GA is a device that continuously measures the amount of two gases (CO<sub>2</sub> and O<sub>2</sub>) from a sample fed at the front of the device with a connected tube (this tube comes from inside of the Scanning Room and ends in the nasal cannula the participant is wearing, as shown in the graph).
+1. **BIOPAC**.
+    The BIOPAC is the main recording hub.
+    It directly receives analogical signals from the Scanning Room (for the case of the RB and the ECG).
+    Indirectly, it receives the analogical signal from the GA, and digital signals from the *Psychopy laptop* ({{ secrets.hosts.psychopy | default("███") }}).
```

Comment:
```suggestion
    Indirectly, it receives the analogical signal from the GA, the triggers from the scanner and digital signals from the *Psychopy laptop* ({{ secrets.hosts.psychopy | default("███") }}).
```

Reply:
It actually doesn't receive the trigger from the scanner, it is just one more signal.